Cargo.toml

```diff
@@ -32,6 +32,9 @@ lnpbp = { git = "https://github.com/lnp-bp/rust-lnpbp", features = ["all"] }
 [build-dependencies]
 configure_me_codegen = "0.3.12"
 
+[patch."https://github.com/lnp-bp/rust-lnpbp"]
+lnpbp = { path = "../rust-lnpbp" }
```

Comment:
Can we move back to the github, otherwise CI will fail on this? We have required PR merged there so now its fine to do that

Reply:
Shit, I avoided committing it the whole time, did it at the end by accident. :D

Reply:
Amended the change in order to avoid making noise in git log.